components/googleMap/googleMap.md

```diff
@@ -0,0 +1,138 @@
+Map:
+
+    initialState = {
+      zoomControl: true,
+      mapTypeControl: true,
+      enableTestCustomStyles: false,
+      enableTestCustomMarker: false,
+      points: [{
+        position: { lat: 52.379189, lng: 4.899431 },
+        customMarker: true,
+        info: {
+          title: 'custom title',
+        },
+      }],
+      center: { lat: 52.379189, lng: 4.899431 },
+      testCustomStyles: [
```

Comment:
@asci I tried to mark that it is only example, not the exact implementation of marker / styles. But np, I removed word `test`, just put '(example)' there.